tests/Feature/CommentTest.php

```diff
@@ -0,0 +1,323 @@
+<?php
+
+namespace Tests\Unit;
+
+use Tests\TestCase;
+use App\Comment;
+use Illuminate\Support\Facades\DB;
+use Illuminate\Foundation\Testing\DatabaseTransactions;
+use App\Member;
+use App\User;
+use Illuminate\Support\Str;
+
+class CommentTest extends TestCase
+{
+    use DatabaseTransactions;
+
+    private $member;
+
+    protected function setUp(): void
+    {
+        parent::setUp();
+        // CommentTest::clearDB();
+        $this->member = Member::findOrFail(2);
+    }
+
+    public function testSeeOwnNormalComment()
+    {
+        $user = User::findOrFail(2);
+        // This doesnt work for some reason
```

Comment:
Vreemd, ik zie niks ongebruikelijks aan deze code

Reply:
Ik denk dat dit een oude comment is, ik zal em weg halen.
Ik weet dat ik liep te kloten met de database en transacties en dat daar wat fout in ging en dat daarom alle DB assertions faalde.

Reply:
Ik heb de comments en oude code weg gehaald.